CMakeLists.txt

```diff
@@ -379,6 +379,7 @@ if(NOT MINGW)
   add_subdirectory(src/metalogger)
   add_subdirectory(src/metarestore)
   add_subdirectory(src/tools)
+  add_subdirectory(src/common/sfserr)
```

Comment:
What about `errors`? I think it's quite clear that the errors are related to SaunaFS. It may be a different matter if it's a shared library (though I think it would be fine if the library was located under some directory labeled SaunaFS, like `/lib/saunafs`) but thankfully it's not.

Reply:
My comment was about completing the word (sfsrror or sfserrors instead of sfserr). I also like only 'errors'. But I don't know how we will use this library in the future. Probably at some point we will link to most of the libraries dynamically. An unique name should not harm. Anyway, this is not a problem, just a personal preference.

Reply:
Personally, if I were including shared library header files, I would much prefer `#include "saunafs/errors.h"` rather than `#include "saunafs/sfserrors.h"`, since I already know I will be dealing with saunafs errors, so if we keep the shared headers in the directory saunafs, it should cause no problems (But it's also a personal preference). As for the library name itself, that can be decided in the CMake file and can be easily changed later if we decide to make a shared library.